### PR TITLE
chore: post-release

### PR DESCRIPTION
**Pull request by henryiii**
## Suggested changelog entry:

```rst

```

### CODE REVIEW

**Comment by henryiii** [comment]
```
RuntimeError: PYBIND11_VERSION_HEX 0x02070000 does not match expected value 0x020701D1!
```

:heart:

**Review by Skylion007 — APPROVED** [review]

**Review by rwgk — APPROVED** [review]
Thanks Henry! — I'll hold off merging #3127 and #2650 until after this PR is merged.

**Pull request merged**
